### ISSUE TITLE
FusedBatchNorm vs FusedBatchNormV3

### ISSUE THREAD

**Issue by slala2121** [tensorflow/tensorflow#42592]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): source
- TensorFlow version (use command below): 1.13.1
- Python version: 3.6
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

I described the issue here: https://stackoverflow.com/questions/63542404/fusedbatchnorm-vs-fusedbatchnormv3

**Comment by slala2121** [comment]
Thanks for your response. Ok, I am using TF 2.2.2 and using v1 (from tensorflow.compat import v1 as v1) for importing the meta graph and still getting same issue.

**Comment by Saduf2019** [comment]
@slala2121
Please share simple indented stand alone code for us to replicate the issue faced, or if possible share colab gist with the error faced.

**Comment by slala2121** [comment]
I posted the code with link to model here:
https://discuss.pytorch.org/t/difference-in-batchnorm-outputs-when-converting-from-tf-model-to-pytorch/93811

**Comment by Saduf2019** [comment]
@slala2121 
I ran the code shared and faced different issue, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/67b52c5eb43404e78824f74a14f173ab/untitled389.ipynb).

**Comment by slala2121** [comment]
@Saduf2019 The model file "model-1.meta" is available here: https://figshare.com/articles/Trained_neural_network_models/8312183

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
Thanks!
